### PR TITLE
Markdown linter corrections.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: hchiramm <hchiramm@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

[skip ci]

### CODE REVIEW

**Comment by humblec** [comment]
The test failure looks to be a spurious one.

**Comment by humblec** [comment]
@jbw976 PTAL :)

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 [![Slack](https://slack.rook.io/badge.svg)](https://slack.rook.io/badge.svg)
 [![Twitter Follow](https://img.shields.io/twitter/follow/rook_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=rook_io&user_id=788180534543339520)
 
-## What is Rook?
+# What is Rook
```
Comment (galexrt) [inline README.md]: The question mark should remain in the headin

Comment (humblec) [reply README.md]: @galexrt brought it back.

File: README.md [hunk 2]
```diff
@@ -44,36 +44,39 @@ Please use the following to reach members of the community:
...
 Convert to your [local timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
 
 Any changes to the meeting schedule will be added to the [agenda doc](https://docs.google.com/document/d/1exd8_IG6DkdvyA0eiTtL2z5K2Ra-y68VByUUgwP7I9A/edit?usp=sharing) and posted to [Slack #announcements](https://rook-io.slack.com/messages/C76LLCEE7/) and the [rook-dev mailing list](https://groups.google.com/forum/#!forum/rook-dev).
 
 Anyone who wants to discuss the direction of the project, design and implementation reviews, or general questions with the broader community is welcome and encouraged to join.
-* Meeting link: https://zoom.us/j/392602367
-* [Current agenda and past meeting notes](https://docs.google.com/document/d/1exd8_IG6DkdvyA0eiTtL2z5K2Ra-y68VByUUgwP7I9A/edit?usp=sharing)
-* [Past meeting recordings](https://www.youtube.com/playlist?list=PLP0uDo-ZFnQP6NAgJWAtR9jaRcgqyQKVy)
+
+- Meeting link: <https://zoom.us/j/392602367>
```
Comment (travisn) [inline README.md]: Is linter recommending to put spaces between the bullet points? It seems better to me without the spaces, but it's a very minor thing

**Comment by humblec** [comment]
@travisn can this PR go in ? :) ..

**Comment by humblec** [comment]
@galexrt Can you or @travisn approve this request ?

**Comment by travisn** [comment]
@humblec did you see [this question](https://github.com/rook/rook/pull/2498#discussion_r250483752)? Just a minor thing...

**Comment by humblec** [comment]
> @humblec did you see [this question](https://github.com/rook/rook/pull/2498#discussion_r250483752)? Just a minor thing...

ah.. sure. addressing it.

File: README.md (hunk 2 above)
Comment (humblec) [reply README.md]: @travisn if you look at this https://github.com/rook/rook/blob/a03c81855bd6deda95c51af423831355ee65a530/README.md , Isnt it looking good compared to without spaces ?

Comment (humblec) [reply README.md]: @travisn its also consistent with other lists in the same README

Comment (travisn) [reply README.md]: The spacing looks different from other lists. For example, under "Contacts", Slack, Forums, Twitter, Email don't have a space, neither do Alpha, Beta, Stable under the project status.

Comment (humblec) [reply README.md]: @travisn PTAL on latest patch set. I have removed the spaces.

Comment (travisn) [reply README.md]: sorry, i wasn't clear on the spaces. I was referring to the extra lines between the items. I'm not normally this picky, but it's for the main readme. :)

Comment (humblec) [reply README.md]: @travisn nw, I have addressed it latest patch set. ptal :)

**Comment by humblec** [comment]
@travisn ptal :)

**Pull request merged**